src/interface/src/app/treatments/direct-impacts/metric-filters/metric-filters.component.ts

```diff
@@ -0,0 +1,158 @@
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+import { MetricSelectorComponent } from './metric-selector/metric-selector.component';
+import { NgFor } from '@angular/common';
+
+export interface Metric {
+  id: string;
+  color: string;
+  label: string;
+}
+
+export const METRICS: Metric[] = [
+  {
+    id: 'ID_CROWN_BULK_DENSITY',
+    color: '',
```

Comment:
I think there are two thing we are trying to combine into one here:
1 - The metric (name/id)
2- The "slot" (position/color)

I dont think metrics should have colors, that comes from the slot on which they are assigned

Reply:
You were right, I updated it